cl/settings/third_party/redis.py

```diff
@@ -1,10 +1,18 @@
 import environ
+from redis import Redis
+from redis.backoff import ExponentialBackoff
+from redis.exceptions import ConnectionError, TimeoutError
+from redis.retry import Retry
 
 env = environ.FileAwareEnv()
 
 
 REDIS_HOST = env("REDIS_HOST", default="cl-redis")
 REDIS_PORT = 6379
+REDIS_MAX_RETRIES = env("REDIS_MAX_RETRIES", default=3)
+REDIS_MAX_DELAY_IN_SECONDS = env("REDIS_MAX_DELAY_IN_SECONDS", default=0.5)
```

Comment:
I think we should start with a greater default `REDIS_MAX_DELAY_IN_SECONDS` according to the backoff exponential policy, since the `REDIS_BACKOFF_BASE` is also 0.5

Reply:
You're absolutely right. Thanks for bringing this to my attention